decidim-proposals/app/controllers/concerns/decidim/proposals/admin/picker.rb

```diff
@@ -2,6 +2,7 @@
 
 require "active_support/concern"
 
+# deprecated
```

Comment:
I would prefer to remove, either to add a ActiveSupport::Deprecation.warn("...") message.